.gitignore

```diff
@@ -119,4 +119,5 @@ venv
 cmd/weaviate/weaviate
 /weaviate
 cmd/weaviate-server/weviate-server
-weaviate-server
\ No newline at end of file
+weaviate-server
+exp/query/testdata/lsmroot/lsm/meta.db
```

Comment:
We need this file for our unit test cases in `query` package. I would rather not ignore it.

Sometime this changes because running just query tests, updates the `meta.db` It's ok to commit it if you see any changes.

Reply:
Ok. as long as it doesn't impact the test results. I'm good 👍 thanks for making sure of that.